IntegrationWithGithub/src/Main.java

```diff
@@ -2,5 +2,7 @@ public class Main {
     public static void main(String[] args) {
 
         System.out.println("This code will be maintained on Github!");
+
+        System.out.println("this line of code was written by clau");
```

Comment:
I don't agree with this line of code, I think we should "bla bla bla"

Reply:
ok, it should work